engines/wage/metaengine.cpp

```diff
@@ -30,6 +30,10 @@
 
 namespace Wage {
 
+void WageEngine::initFeatures() {
```

Comment:
The conventional way is implementing

```C
uint32 WageEngine::getFeatures() {
   return gameDescription->flags;
}
```

and then use this function everywhere.

Reply:
Done